course_info/static/course_info/js/controllers/SearchController.js

```diff
@@ -105,12 +105,24 @@
                 });
             };
 
+            $scope.getCourseDescription = function(course) {
+                // If a course's title is [NULL], attempt to display the short title.
+                // If the short title is also [NULL], display [School] 'Untitled Course' [Term Display]
+                if(typeof course.title != "undefined" && course.title.length > 0){
```

Comment:
Probably a good idea to trim() the value before checking the length, in case it is a string of blank spaces (which won't display a link).